junction/templates/proposals/email/message.html

```diff
@@ -0,0 +1,7 @@
+There is a new comment on "<a href="{{host}}{{proposal.get_absolute_url}}">
+    {{proposal.title}}
+</a>" by <b>{{comment.commenter}}</b>:
+<br/>
+"{{comment.comment}}"
+
+<hr/>
```

Comment:
Add "Please, don't reply to the email. You can login to <url> and leave a comment"